docs/source/gbd2019_models/risk_effects/low_birthweight_short_gestation/index.rst

```diff
@@ -61,6 +61,19 @@ USA analyses were sex-specific.
 category of birthweight and gestational age.**
 [GBD-2019-Risk-Factors-Appendix-LBWSG-Risk-Effects]_
 
+.. note::
```

Comment:
perhaps note this at the end, instead of the beginning.

Reply:
I wanted to put the note right after the first mention of 500g and 2wk categories to clarify that it's an approximate description before the description gets used multiple times. Also, in terms of document structure, the note refers to all the subsequent sections, so I thought it made sense to put it in the intro section of the GBD modeling section. But I guess it could come later if you think that's better.  Where exactly would you put it instead? Maybe at the end of the "Details of Relative Risk Estimation" section? Or in the exposure category/TMREL section since that actually shows the categories?